src/index.js

```diff
@@ -37,14 +44,15 @@ class HCaptcha extends React.Component {
             ...this.props,
             "error-callback"  : this.onerrorCaptcha,
             "expired-callback": this.onerrorCaptcha,
-            "callback"        : this.onsubmitCaptcha
+            "callback"        : this.onsubmitCaptcha,
+            "hl"              : this.languageOverride

```

Comment:
This isn't a valid parameters in the hcaptcha api, its only available as a query parameter on the api url.

Reply:
I've removed this in my commit.

Reply:
Yes, this was only added for testing. It is a slightly cleaner way to pass things in though. We might want to add it to the render args.